ducktape/tests/result.py

```diff
@@ -48,7 +48,7 @@ def __init__(self,
             self.nodes_used = test_context.services.num_nodes()
         else:
             self.services = {}
-            self.nodes_used = 0
+            self.nodes_used = {"": 0}
```

Comment:
Is the key here the OS? If so, shouldn't this string be non-empty? Or just be an empty dictionary?